install.sh

```diff
@@ -12,17 +12,12 @@ function install-theme()
 # Make temp directory
 cd "$(mktemp -d)" || exit 1
 
-# Get the latest release of SFP and unzip it
-mkdir SFP
-cd SFP || exit 1
-wget -q https://github.com/PhantomGamers/SFP/releases/download/0.0.14/SFP_UI-SelfContained-linux-x64.zip
-unzip ./*.zip
-chmod +x SFP_UI
-./SFP_UI
-
 # Install metro
 if [ -d ~/.steam ]
 then
+  echo "WARNING: Removing any previous catppuccin theme installation!"
+  echo "Please cancel the script if you don't want this!"
+  sleep 5
```

Comment:
Please make this a confirmation based thing; as a user, seeing this pop up would be *very* annoying